streampark-console/streampark-console-service/src/main/assembly/script/schema/mysql-schema.sql

```diff
@@ -210,12 +210,19 @@ drop table if exists `t_flink_savepoint`;
 create table `t_flink_savepoint` (
   `id` bigint not null auto_increment,
   `app_id` bigint not null,
+  `job_id`           VARCHAR(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `status` VARCHAR(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT 'COMPLETED',
```

Comment:
I see the checkpoint or savepoint is saved only when `status` is 'COMPLETED', right?

If yes, why need the  `status`?

Reply:
It isn't appropriate  to introduce the field in the Issue.

The platform doesn't collect the failed savepoint/chk into the db.
Maybe it's a good feature to record the exceptionally savepoint/chk for job monitor.